speckle/base/resource.py

```diff
@@ -125,11 +139,42 @@ def _parse_response(self, response, comment=False, schema=None):
     def make_request(self, method, path, data=None, comment=False, schema=None):
         r = self._prep_request(method, path, comment, data)
         resp = self.s.send(r)
+        if not resp.ok:
+            print(resp.text)
+            return None
         response_payload = resp.json()
         assert response_payload['success'] == True, json.dumps(response_payload)
+
         if 'resources' in response_payload:
             return [self._parse_response(resource, comment, schema) for resource in response_payload['resources']]
         elif 'resource' in response_payload:
             return self._parse_response(response_payload['resource'], comment, schema)
         else:
             return response_payload # Not sure what to do in this scenario or when it might occur
+
+    def make_query(self, query):
```

Comment:
There is no need for this method. You can use [`requests` query params](https://requests.readthedocs.io/en/master/user/quickstart/#passing-parameters-in-urls).

Reply:
The question here is how to handle things like `?diameter>10` or just testing for the existence of some parameter like `?properties.beer`. Will read through the `requests` docs to see if there is an illuminating answer...

Reply:
Oooooh... This is a valid point, I forgot speckle uses non-standard query args... Ignore my comments! 😄

Reply:
This is removed and uses the `requests` query params.

Reply:
Whoops, forgot to refresh the page and missed your comment followups 😛 
It's a pretty simple switch, so perhaps we can have a think about it. For now, it works for all the cases that I've been running. Eventually, it would be good to at least have the option of custom query strings.

Reply:
The `requests` params also works just with a plain old query string, so both ways work :)